markdown/nextflow_tutorial.md

```diff
@@ -95,74 +90,81 @@ During this tutorial you will implement a proof of concept of a RNA-Seq pipeline
 ### Step 1 - Command line parameters
 
 The script `rna-ex1.nf` defines the pipeline input parameters that can be defined on the command line but does not contain any processes yet.
-Such parameters follow the convention `params.<name>` in the `run rna-ex1.nf` script file (see line 5-7 for examples).  
+Such parameters follow the convention `params.<name>` in the `run rna-ex1.nf` script file (see line 5-7 for examples).
 
 This is how the parameter definition looks in the script:
-> params.genome = "$baseDir/data/ggal/genome.fa"
+```groovy
+params.genome = "$baseDir/data/ggal/genome.fa"
+```
 
-Because [Nextflow is based on Groovy](https://www.nextflow.io/docs/latest/script.html#language-basics), you will find groovy functions like `println()` in the scripts (see lines 12 to 16).
+Because [Nextflow is based on Groovy](https://www.nextflow.io/docs/latest/script.html#language-basics), you will find groovy functions like `println()` in the scripts (see lines `12` to `16`).
 
 Run `rna-ex1.nf` with the default parameters by using the following command:
 
-```
+```bash
 nextflow run rna-ex1.nf
 ```
 
-To specify a different [input parameter](https://www.nextflow.io/docs/latest/config.html#scope-params), re-define the parameter in the command line by calling the name of the parameter headed by `--`, for example:
+To specify a different [input parameter](https://www.nextflow.io/docs/latest/config.html#scope-params), you can use the parameter in the command prefixed with a double dash (`--`). For example:
 
-```
+```bash
 nextflow run rna-ex1.nf --genome "this/and/that"
 ```
 In this example the path leading to the genome file `"$baseDir/data/ggal/genome.fa"` in the script will be overwritten with `"this/and/that"` when you run the command.
-Note: Parameters that are paths or strings should be in quotes.
+
+> Note: Parameter arguments must be enclosed with quotes if they contain spaces or a file glob pattern
 
 ### Step 2 - Build genome index
 
 `rna-ex2.nf`contains the first [process](https://www.nextflow.io/docs/latest/process.html) called `buildIndex`. It takes the genome file as
 input and creates the genome index by using the `bowtie2-build` tool.
 
 You may recall, that `params.genome` contains only the path to a file.
-In order to access the file we need to evoke the [function `file()`](https://www.nextflow.io/docs/latest/script.html?highlight=file#files-and-i-o) (see line 22 of `rna-ex2.nf`).
+In order to access the file we need to evoke the [`file` method](https://www.nextflow.io/docs/latest/script.html?highlight=file#files-and-i-o) (see line `22` of `rna-ex2.nf`):
+
+```groovy
+genome_file = file(params.genome)
```

Comment:
I felt that this was needed in the docs as without having the tutorial open alongside it was pretty confusing about where `genome_file` came from...